mac/README.md

```diff
@@ -0,0 +1,55 @@
+# melange on mac
+
+Currently `melange` relies on `apk`, which is currently
+not available for mac.
+
+This page documents workarounds to run
+`melange` on a mac.
+
+## OCI Container (Docker)
+
+Use the [official container image](https://github.com/chainguard-images/apko):
```

Comment:
maybe better URL here would be https://github.com/chainguard-images/images/tree/main/images/melange

Reply:
Thank you, good catch @jdolitsky ☝️